2ndmega.html

```diff
@@ -13,10 +13,17 @@
   </style>
   </head>
   
+<
+  <body>   
+  <button><a href="ME.txt">Button Text</a></button>  
+  
+    <centre><h1>Welcome to the other side of my web.</h1></centre>
+
```

Comment:
PLEASE make sure to add <button><a href="2ndmega.html">click here!!</a></button>